plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/helpContexts.xml

```diff
@@ -23,4 +23,16 @@
      The Google Cloud Storage bucket used to stage files associated with the deployment. If this argument is not specified, the application's default code bucket is used.
     </description>
   </context>
+
+  <context id="DeployAppEngineFlexProjectContext" title="App Engine Deployment - Flexible Environment">
+    <!-- Eclipse requires escaped pseudo-HTML here -->
+    <description>Properties used when deploying the project to the App Engine flexible environment.
+
+     &lt;b>App Engine configuration folder&lt;/b>
```

Comment:
The UI uses folder. But based on this: http://stackoverflow.com/questions/5078676/what-is-the-difference-between-a-directory-and-a-folder we should use directory.